bundles/org.eclipse.passage.lic.floating.ecore/model/floating.ecore

```diff
@@ -87,4 +87,14 @@
     <eStructuralFeatures xsi:type="ecore:EReference" name="authentication" ordered="false"
         unique="false" lowerBound="1" eType="#//EvaluationInstructions" containment="true"/>
   </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="GrantAcqisition">
```

Comment:
It is not referenced by other types. How do you plan to store it?

Reply:
It is going to be returned as a root element.

Reply:
ah ok, but we should ensure the corresponding loading code has no strong assumption regarding the file content